lale/lib/autoai_libs/__init__.py

```diff
@@ -102,6 +104,7 @@
 from .float_str2_float import FloatStr2Float as FloatStr2Float
 from .fs1 import FS1 as FS1
 from .fs2 import FS2 as FS2
+from .nsfa import NSFA as NSFA
```

Comment:
In order to solve the import error that results from when the lale.autai_libs module is imported, but the version of autai_libs being used does not have NSFA (as is happening now), this import should be behind a version gate.
An example of getting the version and testing it is here:
https://github.com/IBM/lale/blob/09f4c84140fc45a2488f405130fcf29831fe19bc/lale/lib/autoai_libs/cat_imputer.py#L159